devenv/run_demo.sh

```diff
@@ -83,8 +83,8 @@ fi
 cdvirtualenv src/b2share/devenv
 if [ -n "$REINIT" ]; then
 	echo; echo "### Reinitialize elasticsearch and redis"
-	docker-compose down
-	docker-compose down
+	docker-compose stop
```

Comment:
Not stop, no. We want down so that the elasticsearch and redis containers will be reinitialized completely.